cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -321,6 +321,17 @@
     </description>
   </property>
 
+  <property>
+    <name>system.app.config.dir</name>
```

Comment:
should also add this (without /opt/cdap/master/) to the cdap-site.xml in cdap-standalone module